whisper/config.yaml

```
@@ -14,13 +14,18 @@ backup_exclude:
   - "*.bin"
 options:
   model: tiny-int8
+  custom_model: ""
   language: en
   beam_size: 1
+  debug_logging: false
 schema:
-  model: list(tiny-int8|tiny|base|base-int8|small-int8|small|medium-int8)
+  model: |
+    list(tiny-int8|tiny|tiny.en|base-int8|base|base.en|small-int8|distil-small.en|small|small.en|distil-medium.en|medium-int8|medium|medium.en|large|large-v1|distil-large-v2|large-v2|large-v3|custom)
+  custom_model: str
```

This probably should be an optional option:

```suggestion
  custom_model: str?
```

---

I've set this to optional and added an error case in the run script so the user knows what went wrong if they selected "Custom" for `model` but don't set `custom_model` to anything.